plugins/org.jboss.reddeer.eclipse/src/org/jboss/reddeer/eclipse/topmenu/ExportMenuWizard.java

```diff
@@ -26,25 +26,36 @@
 public abstract class ExportMenuWizard extends WizardDialog implements TopMenuOpenable{
 	
 	protected Matcher<String> matcher;
-	private String[] path;
+	protected String[] wizardPath;
 	
 	/**
-	 * Constructor set path to the specific item in new wizard dialog.
-	 * @param text dialog text
-	 * @param path to the specific item in new wizard dialog
+	 * Constructs new export Wizard that can be found in ImportExportWizard.
+	 * @param shellText export wizard text when next is clicked
+	 * @param wizardCategory export wizard category
+	 * @param wizardName export wizard name
```

Comment:
Would be nice to have examples of wizard category in comment. So the user knows instantly what to use here.